static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-from-google-doc-five-variants.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {
+    makeABTest,
+    makeGoogleDocEpicVariants,
+} from 'common/modules/commercial/contributions-utilities';
+
+const abTestName = 'AcquisitionsEpicFromGoogleDocFiveVariants';
+
+export const acquisitionsEpicFromGoogleDocFiveVariants: EpicABTest = makeABTest(
+    {
+        id: abTestName,
+        campaignId: abTestName,
+
+        start: '2018-04-17',
+        expiry: '2019-06-05',
```

Comment:
Are these the correct set of dates? It seems to wide

Reply:
yep - I don't want them to accidentally expire - we've had problems with that in the past